spec/aws/session_store/dynamo_db/garbage_collection_spec.rb

```diff
@@ -1,16 +1,3 @@
-# Copyright 2013 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"). You
-# may not use this file except in compliance with the License. A copy of
-# the License is located at
-#
-#     http://aws.amazon.com/apache2.0/
-#
-# or in the "license" file accompanying this file. This file is
-# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
-# ANY KIND, either express or implied. See the License for the specific
-# language governing permissions and limitations under the License.
-
 require 'spec_helper'
 
 describe Aws::SessionStore::DynamoDB::GarbageCollection do
```

Comment:
This can be removed entirely now right?

Reply:
I forgot about tests - but yes I can delete those. I debated whether the garbage collection was useful even with the TTL feature. Thoughts?

Reply:
Yeah - I think it makes sense to remove the garbage collection.  I think using TTL here is a better practice and we should encourage that.  Side note: we may want to add options for using ddb encryption.  I'd also potentially like to evaluate the pessimistic locking.... I'm not quite sure I understand the case for the fairly expensive pessimistic locking vs using a straightforward optimistic locking strategy.  But thats out of scope for this change.

Reply:
I think it makes sense to remove pessimistic locking as well.